dateparser/date_parser.py

```diff
@@ -2,14 +2,15 @@
 from __future__ import unicode_literals
 
 import re
-
 from datetime import datetime
+from six import binary_type
```

Comment:
I see, thank you. We had long ago quick discussion on the topic [here](https://github.com/scrapinghub/dateparser/pull/15#discussion-diff-21075948), but as python3 support is still pending, let's for now use line that i've suggested.